mitodl/ocw-hugo-themes
# fix: sentry DSN configured via env

#### Pre-Flight checklist


#### What are the relevant tickets?
closes https://github.com/mitodl/ocw-hugo-themes/issues/881

#### What's this PR do?
- Configures Sentry DSN from env.
- If `SENTRY_DSN` env is not found or not set, then exceptions are ignored/not logged to sentry.

#### How should this be manually tested?
- Checkout this branch
- Generate errors locally (how to do it? Please check the description of the issue ticket)
- Verify that these errors are not logged to Sentry.
- Now set `SENTRY_DSN` in your .env 
- Rebuild.
- Repeat the above steps to generate errors locally.
- Verify that these errors are logged to Sentry.
- Test any other use case that needs to be tested in order to verify that these changes have no side-effects
